### PR TITLE
Adds an update param

### PR DESCRIPTION
**Pull request by eddysims**
I really enjoy the tool, but I found it a little bit challenging to work with tools like browser sync.

I have added in an option to allow you to pass in an `--update` param to your script that will update a passed in file with a random string once the watch has finished uploading. It seems to be much more reliable then the solutions presented in the `slate` repo.

example:

```
scripts: {
  ...,
  "watch": "qs theme watch --target YourTarget --update './src/update.txt'",
  "browserSync": "browser-sync start --proxy 'https://mystoreurl.myshopify.com/' --files './src/update.txt' --config bs-config.js",
}
```

or:

```
scripts: {
  ...,
  "watch": "qs theme watch --target YourTarget --update './src/update.txt'",
}
```

```
const path = require("path");
const browserSync = require("browser-sync").create();

browserSync.init({
  files: `${path.join(__dirname, "./src/update.txt")}`,
  proxy: 'https://mystoreurl.myshopify.com/',
  port: 8000,
  snippetOptions: {
    rule: {
      match: /<head[^>]*>/i,
      fn(snippet, match) {
        return match + snippet;
      },
    },
  },
});
```

### CODE REVIEW

**Comment by joshuaiz** [comment]
Thanks @eddysims! Great solution. Would love if this could be merged. It really would make things easier.

**Comment by eddysims** [comment]
@internalfx not sure if this project is still being maintained. But this PR is quite old. Just pinging to let you know that its here.

**Comment by internalfx** [comment]
Yes, it's being maintained. Hopefully I can get to this soon.

**Comment by eddysims** [comment]
@internalfx just checking in on this again

**Comment by eddysims** [comment]
@internalfx just checkin in again

**Comment by internalfx** [comment]
@eddysims Does browser sync just watch a file for changes and then refresh the page? I personally have never used it.

**Comment by hughker** [comment]
@internalfx — good to see ya! 

**TL;DR**: yes, Browsersync watches a directory for any changes then refreshes the page after x-seconds.

---

Here is the Browsersync command that I run in another terminal after I start quickshot:

```shell
browser-sync start --https --proxy "https://example.com" --files "theme/**/*" --reload-delay "3000" --config bs-config.js
```

So, I'm telling it to proxy my liveStore URL; watch all files within the `theme/` directory; if any files changed, it'll wait 3s then refresh my page; and, the last bit it just to fix the issue with Shopify inserting their tracking scripts into `<head>` which makes Browsersync get wonky.

---

Here is the content of `bs-config.js` in case someone finds this and is curious:

```js
module.exports = {
  snippetOptions: {
    rule: {
      match: /<\/head>/i,
      fn: function(snippet, match) {
        return snippet + match;
      }
    }
  }
};
```

_Note: I found this on some other issue(s) — if I find it I will credit accordingly._

**Comment by hughker** [comment]
@eddysims — this is seems to be more tailored to a Shopify Slate Theme/Workflow: rather than a Universal Shopify Theme setup, no? If I remember correctly, Slate used to have an option `slate watch` that would initialize a Browsersync session. Does that not work anymore?

Also, I'm curious how you're using Slate with quickshot as I thought they handle all the uploading for changed files etc. 

Look forward to hearing back from you and learning more about your specific setup!

/cc @joshuaiz — pinging you too in case you have some more insight to add as you have commented on this in the past.

**Comment by internalfx** [comment]
Thanks @hughker, I'm updating huge amounts of the QS codebase. Node has changed a lot in just a few years!

Hopefully, I'll have something released this weekend.

**Comment by internalfx** [comment]
@eddysims Closed with commit bf65d81
